tests/models.py

```diff
@@ -1,13 +1,12 @@
-from __future__ import absolute_import, unicode_literals
-
 from django.db import models
-
 from modelcluster.fields import ParentalKey
 
 try:
-    from wagtail.core.models import Page, Orderable
-except ImportError:  # fallback for Wagtail <2.0
-    from wagtail.wagtailcore.models import Page, Orderable
+    from wagtail.models import Page
+except ImportError:  # fallback for Wagtail <5.0
```

Comment:
This fallback is not required - all supported versions recognise `wagtail.models`.